Vagrantfile

```diff
@@ -0,0 +1,29 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+
+Vagrant.configure(2) do |config|
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+
+  # create intellij node (change this to new project)
+    
+ config.vm.define :intellij do |intellij_config|
+
+     #intellij_config.vm.box = "bento/centos-6.7"
+     intellij_config.vm.box = "cyplo/centos-6.5-gui"
```

Comment:
Since we are no longer running the bash script, can we switch this back to bento/centos-6.7?